charts/federator/templates/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: federator
+  labels:
+    wireService: federator
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: ClusterIP
+  ports:
+    # TODO: Do we name it something else to get the _correct_ SRV record?
```

Comment:
I don't quite understand this TODO. Naming this port http seems fine to me. Do we use the port name somewhere else?

Reply:
When we call this port `http`, kubernetes will create SRV record as `_http._tcp.federator.<namespace>.cluster.local`.
I assume for discovering remote federators, we will rely on an SRV record like `_wire-federator._tcp.<domain>`.
So, if we rename this port from http to `wire-federator` and configure domain as `federator.<namespace>.cluster.local` we will be able to test federation in same kubernetes cluster without having to mess with global DNS records. All of this is obviously not thought through, so I would much rather tackle it later. I will remove this TODO and write this information in some issue in JIRA.